packages/runtime/container-runtime/src/index.ts

```diff
@@ -5,7 +5,23 @@
 
 export {
 	ContainerMessageType,
+	UnknownContainerMessageType,
 	ContainerRuntimeMessage,
+	LocalContainerRuntimeMessage,
+	InboundContainerRuntimeMessage,
+	OutboundContainerRuntimeMessage,
+	ContainerRuntimeAliasMessage,
+	ContainerRuntimeBlobAttachMessage,
+	ContainerRuntimeChunkedOpMessage,
+	ContainerRuntimeDataStoreOpMessage,
+	ContainerRuntimeRejoinMessage,
+	InTransitContainerRuntimeIdAllocationMessage,
+	InboundContainerRuntimeAttachMessage,
+	UnknownContainerRuntimeMessage,
+	InboundSequencedContainerRuntimeMessage,
+	LocalContainerRuntimeIdAllocationMessage,
+	OutboundContainerRuntimeAttachMessage,
+	OutboundSequencedContainerRuntimeMessage,
 	IContainerRuntimeMessageCompatDetails,
```

Comment:
Looks like I need to introduce this project to `type` imports including an eslint plugin that will fix and enforce them.